src/libmongoc/tests/test-mongoc-cluster.c

```diff
@@ -1231,10 +1229,14 @@ auto_ismaster (request_t *request, void *data)
    dollar_query_test_t *test;
```

Comment:
Rename this function to `auto_hello`?

Reply:
Has been changed to `auto_hello_callback` during refactoring.